lib/cloud.js

```diff
@@ -1,23 +1,43 @@
-var path = require('path')
-var hyperarchiver = require('hyperarchiver')
+var assert = require('assert')
+var fs = require('fs')
+var memdb = require('memdb')
+var Archiver = require('hypercore-archiver')
+// var ArchiverServer = require('archiver-server') TODO needed?
```

Comment:
We need some sort of server to connect to the dat swarms. `hypercore-archiver` only acts as storage, it doesn't connect to peers itself.

We could also implement the server directly with discovery-swarm but think archiver-server is good for this use case. We don't need to use the HTTP parts.